charts/prometheus-rabbitmq-exporter/values.yaml

```diff
@@ -71,9 +71,69 @@ prometheus:
+## The value is evaluated as a template, so, for example, the value can depend on .Release or .Chart
+## ref: https://github.com/coreos/prometheus-operator#customresourcedefinitions
+prometheusRule:
+  enabled: false
+  additionalLabels: {}
+  namespace: ""
+  rules: []
+    ## These are just examples rules, please adapt them to your needs.
+    ## Make sure to constraint the rules to the current service.
+    #  - alert: RabbitmqNodeDown
+    #    expr: rabbitmq_running{service="{{ template "prometheus-rabbitmq-exporter.fullname" . }}"} == 0
+    #    for: 5m
+    #    labels:
+    #      severity: warning
+    #    annotations:
+    #      summary: A Rabbitmq node is down
+    #      description: |
+    #        The Rabbitmq node {{ "{{ $labels.node }}" }} of
+    #          the cluster tracked by {{ "{{ $labels.service }}" }} was not running during the last 5m.
+    #  - alert: RabbitmqClusterDown
+    #    expr: |
+    #      rabbitmq_up{service="{{ template "prometheus-rabbitmq-exporter.fullname" . }}"} == 0
+    #    for: 5m
+    #    labels:
+    #      severity: critical
+    #    annotations:
+    #      summary: The Rabbitmq cluster {{ "{{ $labels.service }}" }} is maybe down.
+    #      description: |
+    #          The Rabbitmq exporter couldn't scrape any Rabbitmq node of the
+    #          cluster tracked by {{ "{{ $labels.service }}" }} during the last 5m, the cluster is maybe down.
+    #  - alert: RabbitMQClusterPartition
+    #    expr: rabbitmq_partitions{service="{{ template "prometheus-rabbitmq-exporter.fullname" . }}"} > 0
+    #    for: 5m
+    #    labels:
+    #      severity: critical
+    #    annotations:
+    #      summary: A cluster partition was detected
+    #      description: |
+    #          Cluster partition in Rabbitmq cluster tracked by {{ "{{ $labels.service }}" }} was detected
+    #          by the node {{ "{{ $labels.node }}" }}
+    #  - alert: RabbitmqOutOfMemory
+    #    expr: |
+    #      rabbitmq_node_mem_used{service="{{ template "prometheus-rabbitmq-exporter.fullname" . }}"}
+    #      / rabbitmq_node_mem_limit{service="{{ template "prometheus-rabbitmq-exporter.fullname" . }}"}
+    #      * 100 > 90
+    #    for: 5m
+    #    labels:
+    #      severity: warning
+    #    annotations:
+    #      summary: The Rabbitmq node {{ "{{ $labels.node }}" }} is Out of memory
+    #      description: |
+    #          Memory available for Rabbmitmq node {{ "{{ $labels.node }}" }} is lower than 10%
+    #  - alert: RabbitmqTooManyConnections
+    #    expr: rabbitmq_connectionsTotal{service="{{ template "prometheus-rabbitmq-exporter.fullname" . }}"} > 1000
+    #    for: 5m
+    #    labels:
+    #      severity: warning
+    #    annotations:
+    #      summary: Too many connections to the Rabbitmq cluster
+    #      description: |
+    #          The Rabbitmq cluster tracked by {{ "{{ $labels.service }}" }} has too many connections {{ "{{ $value }}" }} (> 1000)
```

Comment:
@denny60004 
Line 83 to 136 has to be uncommented; otherwise it's not the same as before upgrade as stated in README.md.